src/electron-forge-start.js

```diff
@@ -25,6 +26,7 @@ import { start } from './api';
     dir,
     interactive: true,
     enableLogging: program.enableLogging,
+    appPath: program.appPath,
```

Comment:
Ah, yes, you are right. Sorry, @MarshallOfSound, sloppy of me.

I did some more detailed testing, and it looks like the start command's command parsing is pretty broken. None of the command-line options will ever get parsed because [this](https://github.com/electron-userland/electron-forge/blob/master/src/electron-forge-start.js#L22) line slices it down to `[ '<path>/node', '<path>/electron-forge-start' ]`. So any command-line arguments meant for the start command itself will not be handled, and instead will get [passed through](https://github.com/electron-userland/electron-forge/blob/master/src/electron-forge-start.js#L28) to the electron application itself.

It looks like the fix would be to change the `dir` parameter to be an option (`-d dir`) so we can tell it apart from the `args` to be passed to the electron app. I'd be happy to fix this in my PR and get some unit tests in place, but being a new contributor to this project, I'd like a nod from somebody already involved first.